app/assets/stylesheets/pages/_home.scss

```diff
@@ -20,23 +19,20 @@
 
   h3 {
     font-size: 30px;
-    color: white;
-    font-family: $body-font;
+    color: $light-gray;
   }
```

Comment:
$light-gray is one of Bootstrap's variable.
The one we use is `color: lightgrey;`, I'm not sure they're exactly the same so just in case use this one instead please